### PR TITLE
Ensure that all Cassandra queries can run

### PR DESCRIPTION
**Pull request by c-w**
Implementation is terrible, but this is likely the best we can do with the current schema.

### CODE REVIEW

File: src/resolvers-cassandra/Edges/queries.js [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
+      clauses.push('(event_time <= ?)');
+      params.push(args.toDate);
+    }
+
+    if (pipeline) {
+      clauses.push('(pipeline = ?)');
+      params.push(pipeline);
+    }
+
+    let query = `SELECT detectedkeywords FROM fortis.events WHERE ${clauses.join(' AND ')} ALLOW FILTERING`;
```
Comment (jcjimenez) [inline src/resolvers-cassandra/Edges/queries.js:38]: `ALLOW FILTERING` ouch!

File: src/resolvers-cassandra/Facts/queries.js [hunk 2]
```diff
@@ -23,26 +24,24 @@ function cassandraRowToFact(row) {
...
 
-    clauses.push('(detectedkeywords CONTAINS ?)');
-    params.push(keyword);
+    if (keywordAndPipeline.b) {
+      clauses.push('(pipeline = ?)');
+      params.push(keywordAndPipeline.b);
+    }
 
-    const query = `SELECT * FROM fortis.events WHERE (${clauses.join(' AND ')})`;
+    const query = `SELECT * FROM fortis.events WHERE ${clauses.join(' AND ')} ALLOW FILTERING`;
```
Comment (jcjimenez) [inline src/resolvers-cassandra/Facts/queries.js:44]: Another ouch. AFAIK, `ALLOW FILTERING` fetches everything and does driver-side filtering, right?

**Review by jcjimenez — APPROVED** [review]
I think this will work as long as we have a very small number of records.

File: src/resolvers-cassandra/Edges/queries.js (hunk 1 above)
Comment (c-w) [reply src/resolvers-cassandra/Edges/queries.js:38]: Yes, the schema right now requires us to do this :( We really have to formulate the v2 queries that we want and make sure that the DB schema supports them. Adding @erikschlegel for visibility.

File: src/resolvers-cassandra/Facts/queries.js (hunk 2 above)
Comment (c-w) [reply src/resolvers-cassandra/Facts/queries.js:44]: See [comment above](https://github.com/CatalystCode/project-fortis-services/pull/73#discussion_r126229285).

**Pull request merged**